### PR TITLE
Feature | Add Queue Tries

### PR DESCRIPTION
**Pull request by hamed-aloware**
- Add Queue Tries

### CODE REVIEW

File: src/FairSignalJob.php [hunk 1]
```diff
@@ -93,6 +97,23 @@ public function handle()
...
+
+        // get queue tries from fair-queue config
+        $tries = config("fair-queue.queues.{$queue}.tries");
+
+        // sanity check
+        if(!$tries) {
+            $tries = config('fair-queue.queues.default.tries');
+        }
+
+        return $tries;
```
Comment (madagon) [inline src/FairSignalJob.php:114]: can be simplified as
`
return config("fair-queue.queues.{$queue}.tries") ?? config('fair-queue.queues.default.tries') ?? 1;
`

**Review by madagon — CHANGES_REQUESTED** [review]
config file changes missing?

**Comment by hamed-aloware** [comment]
> 

Added

**Comment by hamed-aloware** [comment]
> config file changes missing?

Added

File: src/FairSignalJob.php (hunk 1 above)
Comment (hamed-aloware) [reply src/FairSignalJob.php:114]: Yes but it won't be that readable I think

**Review by mohsen-aloware — APPROVED** [review]

**Pull request merged**